src/server/migrations/20200510015322_create_modules_schema.js

```diff
@@ -1,18 +0,0 @@
-// TODO: This is a sample migration for demonstration. Remove before adding real ones.
-
```

Comment:
See here for the discussion about the modules tables #18

Either remove the backend endpoint as well or keep everything including migrations and seeds